src/main/java/com/example/Customer.java

```diff
@@ -0,0 +1,14 @@
+package com.example;
+
+/**
+ * A customer is a person who visits a shop for buying goods.
+ * <p>
+ * The customer can have a loyalty card that has a specific number.
```

Comment:
It should be `<p>The customer...`. This is a newer style. Older Java code uses `<p>` as a paragraph delimiters. When we apply modern static code analyzers like Error Prone or Check Style, we'd spot this automatically. For now, please put the paragraph marks before the first character of the text that follows.